src/commands/CmdSync.cpp

```diff
@@ -78,23 +83,28 @@ int CmdSync::execute (std::string& output)
     std::ostringstream os;
     os << "GCP bucket " << gcp_bucket;
     server_ident = os.str();
-  } else if (origin != "") {
+  } else if (server_url != "") {
```

Comment:
To make things easier for users, let's automatically convert `sync.server.origin` to a URL, if `sync.server.url` is not set.

Reply:
Sounds great! - but I'm not sure what you mean exactly by "convert sync.server.origin to a URL"?

I think both `sync.server.url` and `sync.server.origin` are strings in this context and I think I already made it so if `sync.server.url` is not set and `sync.server.origin` is set we use `sync.server.origin` as the 'URL'.

Reply:
I meant what you've done above on line 73:
```c++
std::string server_url = url == "" ? origin : url;
```
If that was already present when I made the comment, I apologize!